mercury-wallet/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: mercury-wallet_web_1
+      APP_PORT: 3000
+      
+  web:
+    image: layer2tech/mercury-wallet-ui:latest
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - 3000:3000

```

Comment:
This is not required because our App Proxy (which protects apps with the user's password and 2FA) will open your app's port (defined in `umbrel-app.yml`).

Reply:
Just to clarify, I should remove the port mapping config for the container?

Reply:
If this is the frontend (web app) of Mercury wallet, then all requests would go through the Umbrel App Proxy which forwards the requests to this container. This configuration: https://github.com/getumbrel/umbrel-apps/pull/366/files#diff-70c92909bde90f135ea8b99a8546b53d8b5481c3152700c3aba0cc18a3f6f876R5-R7 tells the App Proxy where it should send requests to.